zerver/webhooks/slack/doc.md

```diff
@@ -21,6 +21,9 @@ See also the [Slack-compatible webhook](/integrations/doc/slack_incoming).
    section and make sure **Bot Token Scopes** includes:
    `channels:read`, `channels:history`, `users:read`, `eomji:read`,
    `team:read`, `users:read`, `users:read.email`.
+   > ***Note**: If you worry about the scopes that we're asking, refer to
+   [Related Documentation](#related-documentation) section for more context
+   on the Slack APIs that we use.*
```

Comment:
Use the warning syntax here. And let's have all the doc updates in one commit.

Also, I'd say "If you're concerned about the scopes specified above, see Slack's Events API documentation for more context." with a direct link here.